Core/src/org/sleuthkit/autopsy/casemodule/events/ContentTagAddedEvent.java

```diff
@@ -50,7 +53,43 @@ public ContentTagAddedEvent(ContentTag newTag, ContentTag deletedTag) {
      * @throws NoCurrentCaseException
      * @throws TskCoreException
      */
+    @Override
     ContentTag getTagByID() throws NoCurrentCaseException, TskCoreException {
         return Case.getCurrentCaseThrows().getServices().getTagsManager().getContentTagByTagID(getTagID());
     }
+    
+    /**
+     * Returns the list of tags that were removed as a result of the addition 
+     * of the ContentTag.
+     * 
+     * @return A list of removed tags or null if no tags were removed.
+     */
+    @SuppressWarnings("unchecked")
+    public List<DeletedContentTagInfo> getRemovedTags() {
+        Object oldValue = this.getOldValue();
```

Comment:
An instanceof is not enough. I even tried first an instancof List then getting the first item and checking that it is a DeletedContentTagInfo and the warning still appears.
https://stackoverflow.com/questions/10108122/how-to-instanceof-listmytype

Reply:
Bummer. Let's bite the bullet then and treat ourselves to a dedicated field, instead of using the super class oldValue field. Then we can override getOldValue and we can also provide a properly typed method to return the List. This is the alternate implementation strategy that I mentioned earlier - it appears we can use it here to avoid the warning suppression.